### ISSUE TITLE
Any command I use doesn't work because of some update password thing

### ISSUE THREAD

**Issue by KV2808** [sanjaybaskaran01/Valorina#17]
I signed in and gave my information to the Valorina bot.

However any time I run a command, ex. '+store kvlx na' the bot gives me an update password message even though I didn't change any of my information.
<img width="468" alt="Screen Shot 2022-03-19 at 1 47 05 PM" src="https://user-images.githubusercontent.com/81344942/159137873-ffa87fc5-5a00-4e63-aa73-4fca96e9a1db.png">

**Comment by sanjaybaskaran01** [comment]
Hey @KV2808 , did you try updating your password in valorina?

**Comment by KV2808** [comment]
> Hey @KV2808 , did you try updating your password in valorina?

Yes I tried changing my password multiple times but it still gave me the same error.

**Comment by Choyuin** [comment]
> I signed in and gave my information to the Valorina bot.
> 
> However any time I run a command, ex. '+store kvlx na' the bot gives me an update password message even though I didn't change any of my information.

I'm having the same issue rn, I still don't know how to fix it.
